src/index.js

```diff
@@ -81,3 +96,12 @@ todoInput.addEventListener('keypress', (e) => {
     }
   }
 });
+
+clearAllBtn.addEventListener('click', () => {
+  todoList.todoTasks.filter(() => {
+    todoList.todoTasks = [];
+    return todoList.todoTasks;
+  });
+  setLocalStore(todoList.todoTasks);
+  displayTasks();
+});
```

Comment:
- [ ] Please your `Clear all completed button` clears all the `To do` list(`tasks`) instead of clearing `only the completed(checked) tasks`.  Please kindly fix that, as is a requirement for for this project. Please make sure the `Clear all completed button` clears only the tasks marked(`checked`) as completed not all the tasks.

Reply:
Thank you! I didn't understand the requirement.